index.bs

```diff
@@ -6530,7 +6584,7 @@ strong [=/map=] from handle ids to their corresponding objects.
 script.InternalId = text;
 </pre>
 
-The <code>script.InternalId</code> type represents the id of 
+The <code>script.InternalId</code> type represents the id of
```

Comment:
let's split editorial changes into a separate CL for easier review

Reply:
This is just my text editor removing whitespace on save. This keeps coming up because this repository has no linter. I'll look into adding pre-commit here.

For now, could you just ignore this diff? It's really small. I am happy to split it before submission.

Reply:
https://github.com/w3c/webdriver-bidi/pull/603